guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,49 @@
+We'd like to thank the following people who contributed to the Foreman {{page.version}} release:
+
+Adam Lazik,
+Adam Růžička,
+Aneta Šteflová Petrová,
+Archana Kumari,
+Bastian Schmidt,
+Ben Erickson,
+Bernhard Suttner,
+Brenden Wood,
+Chris Roberts,
+Cole Higgins,
+Eric Helms,
+Evgeni Golov,
+Ewoud Kohl van Wijngaarden,
+Francesco Di Nucci,
+Gaurav Talreja,
+Girija Soni,
+Hao Yu,
+Ian Ballou,
+Jan Löser,
+Jeremy Lenz,
+Leos Stejskal,
+Lucy Fu,
+Maria Agaphontzev,
+Markus Bucher,
+Markus Reisner,
+Martin Alfke,
+Martin Spiessl,
+Matthew Davis,
+Maximilian Kolb,
+Nadja Heitmann,
+Nofar Alfassi,
+Oleh Fedorenko,
+Partha Aji,
+Patrick Creech,
+PopiBrossard,
+Samir Jha,
+Sayan Das,
+Shimon Shtein,
+Takashi Kajinami,
+Tim Meusel,
+VHS,
+akumari,
```

Comment:
This is redundant because Archana is already listed. This was also an issue with 3.13. https://github.com/theforeman/theforeman.org/pull/2218 fixes the script.